domain-models-runtime/src/main/java/org/folio/rest/tools/utils/OutStream.java

```diff
@@ -13,6 +17,17 @@
 
   private Object data;
 
+  public OutStream() {
+  }
+
+  public OutStream(RowSet<Row> result) {
+    RowIterator<Row> it = result.iterator();
+    StringBuilder s = new StringBuilder();
+    while (it.hasNext()) {
+      s.append(it.next().toString());
```

Comment:
This does not export the column names and therefore fails the new tests in AdminAPI.
Expected valid JSON: `[{"relname":"users","buffered":"2320 kB","buffers_percent":"0.4","percent_of_relation":"100.3"}, {"relname":"pg_depend_reference_index","buffered":"600 kB","buffers_percent":"0.1","percent_of_relation":"31.4"}]`
Actual invalid JSON: `[users, 2320 kB, 0.4, 100.3][pg_depend_reference_index, 600 kB, 0.1, 31.4]`

Reply:
Yes. Saw that and should be fixed by now